### PR TITLE
DEV9: add ATA Idle Immediate (E1) stub

### PR DESCRIPTION
**Pull request by macmenot**
Fixes https://github.com/PCSX2/pcsx2/issues/4358

### CODE REVIEW

File: pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp [hunk 1]
```diff
@@ -230,3 +230,11 @@ void ATA::HDD_Idle()
 	DevCon.WriteLn("HDD_Idle for %is", idleTime);
 	PostCmdNoData();
 }
+
+void ATA::HDD_IdleImmediate()
+{
+	if (!PreCmd())
+		return;
+	DevCon.WriteLn("HDD_IdleImmediate");
```
Comment (lightningterror) [inline pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp:238]: I'd add `DEV9:` as a prefix for the log.

Comment (macmenot) [reply pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp:238]: @lightningterror I can add that, though none of the other calls to `DevCon.WriteLn(…)` in ATA_CmdNoData.cpp currently have that prefix, so do you want me to add it to each of them under this PR?

Comment (lightningterror) [reply pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp:238]: Yeah, tho maybe in a separate commit.

Comment (macmenot) [reply pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp:238]: @lightningterror I'll go ahead and do the whole ./DEV9/... tree for consistency and push it up as a follow-on PR

Comment (macmenot) [reply pcsx2/DEV9/ATA/Commands/ATA_CmdNoData.cpp:238]: @lightningterror I'll open up a follow-on PR with this commit https://github.com/macmenot/pcsx2/commit/db839213f1b09c5f0669310dec79cacb6af458dc once this PR has been approved and merged

**Pull request merged**